<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>THORViz</class>
 <widget class="QMainWindow" name="THORViz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1239</width>
    <height>799</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>THORViz</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="100,1">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="9,0">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="ThorVizWidget" name="vizGL">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="rightMargin">
             <number>19</number>
            </property>
            <item>
             <widget class="QRadioButton" name="show_data_radiobutton">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>&amp;Data</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="show_grid_radiobutton">
              <property name="text">
               <string>&amp;Grid</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="show_wireframe_checkbox">
              <property name="text">
               <string>Wireframe</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>5</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <property name="leftMargin">
               <number>5</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Scalar</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="scalar_display_combobox"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <property name="leftMargin">
               <number>5</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Vector</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="vector_display_combobox"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="rightMargin">
             <number>8</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Time</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="animation_slider">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="animation_spinbox">
              <property name="maximum">
               <number>1000</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1000,0">
        <property name="rightMargin">
         <number>13</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Level</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="level_slider">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="level_spinbox"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1239</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZero_Position"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionZero_Position">
   <property name="text">
    <string>&amp;Zero Position</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ThorVizWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>thorvizwidget</header>
   <slots>
    <slot>set_image(int)</slot>
    <slot>show_data(bool)</slot>
    <slot>show_grid(bool)</slot>
    <slot>show_wireframe(bool)</slot>
    <slot>set_level(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>animation_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>vizGL</receiver>
   <slot>set_image(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>653</x>
     <y>738</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>animation_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>animation_spinbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>758</x>
     <y>738</y>
    </hint>
    <hint type="destinationlabel">
     <x>1145</x>
     <y>743</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>show_grid_radiobutton</sender>
   <signal>toggled(bool)</signal>
   <receiver>vizGL</receiver>
   <slot>show_grid(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>738</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>show_data_radiobutton</sender>
   <signal>toggled(bool)</signal>
   <receiver>vizGL</receiver>
   <slot>show_data(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>711</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>show_wireframe_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>vizGL</receiver>
   <slot>show_wireframe(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>765</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>show_data_radiobutton</sender>
   <signal>toggled(bool)</signal>
   <receiver>THORViz</receiver>
   <slot>show_data(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>711</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>level_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>level_spinbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1182</x>
     <y>475</y>
    </hint>
    <hint type="destinationlabel">
     <x>1218</x>
     <y>768</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>level_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>vizGL</receiver>
   <slot>set_level(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1182</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>level_spinbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>level_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1190</x>
     <y>755</y>
    </hint>
    <hint type="destinationlabel">
     <x>1175</x>
     <y>599</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>animation_spinbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>animation_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1104</x>
     <y>727</y>
    </hint>
    <hint type="destinationlabel">
     <x>997</x>
     <y>723</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scalar_display_combobox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>THORViz</receiver>
   <slot>select_scalar_display(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>721</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>768</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vector_display_combobox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>THORViz</receiver>
   <slot>select_vector_display(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>756</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>777</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>show_wireframe(bool)</slot>
  <slot>show_data(bool)</slot>
  <slot>show_grid(bool)</slot>
  <slot>select_scalar_display(QString)</slot>
  <slot>select_vector_display(QString)</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
